Update secure-scan.yml for ESRP Malware Scan Azure Pipelines

kmyrup opened the pull request:
Adding in ESRP malware scanning task

kmyrup committed:
Update secure-scan.yml for ESRP Malware Scan Azure Pipelines

Adding in ESRP malware scanning task

kmyrup committed:
Update secure-scan.yml for ESRP Malware Scan Azure Pipelines

Adding in ESRP malware scanning task

kmyrup committed:
Update secure-scan.yml for ESRP Malware Scan Azure Pipelines

Adding in ESRP malware scanning task

kmyrup committed:
Merge branch 'master' into azure-pipelines-ESRP

kmyrup committed:
Update secure-scan.yml

silveyy commented on pipelines/secure-scan.yml:
```diff
@@ -34,6 +34,17 @@ steps:
+    displayName: 'Run ESRP Malware Scan'
+    inputs:
+      ConnectedServiceName: 'ESRP Malware Scanning service connection'
+      FolderPath: '$(Build.SourcesDirectory)\objects'
```
>objects [](start = 45, length = 7)

can we only scan the changed folder?

kmyrup replied:
Until we can detect the changed folder reliably, I'd rather scan them all.

kmyrup committed:
Change timeout to 60 minutes to ensure robustness

To ensure we don't time out, moving to 60 minutes.

kmyrup committed:
Update secure-scan.yml

add only to scan when there are console extensions contributed

kmyrup committed:
Update ExtensionDownloader.ps1

Adding env var creation and file copy to staging folder, for use by ESRP scanning task.

silveyy commented on tools/ExtensionDownloader.ps1:
```diff
@@ -164,13 +164,16 @@ function DownloadAndExpand
+        Write-Host "##vso[task.setvariable variable=codeSignEnabled]true"
+        
+        $repoRootFolder = (get-BuildRootDirectory);
+        $consoleExtensionFolder = $reporepoRootFolder + "\objects\ConsoleExtension\"
```
>repo [](start = 39, length = 4)

typo

kmyrup committed:
Update secure-scan.yml

Consume ESRPScanFolder variable.

kmyrup committed:
Update ExtensionDownloader.ps1

fixed typo

silveyy reviewed (dismissed):
:shipit:

kmyrup committed:
testing edit of an extension to test pipeline

testing edit of an extension to test pipeline - will revert after

kmyrup committed:
reverting test change

reverting test change

silveyy reviewed (approved):
:shipit:

kmyrup committed:
Update Right Click Twos.json

fix revert

kmyrup committed:
fix the revert whitespace

get rid of whitespace diff
